src/index.html

```diff
@@ -10,8 +10,185 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
     <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form action="ttps://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="group">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label>
+          Surname:
+            <input
+          autocomplete="off"
+          type="text"
+          name="surname"
+          >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          Name:
+            <input
+          autocomplete="off"
+          type="text"
+          name="name"
+          >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          How old are You?
+            <input
+          type="number"
+          name="age"
+          value="12"
+          min="1"
+          max="100"
+          >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          Full date of birth:
+            <input type="date" name="birthday">
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          I accept the term of the agreement
+            <input
+          type="checkbox"
+          name="terms"
+          required
+          >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label>
+          E-mail:
+            <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
```

Comment:
```suggestion
            <input
              type="email"
              name="email"
              placeholder="email@example.com"
            >
```
[code style](https://mate-academy.github.io/style-guides/htmlcss.html)